Code/Editor/EditorViewportSettings.h

```diff
@@ -51,6 +51,9 @@ namespace SandboxEditor
     SANDBOX_API bool ShowingGrid();
     SANDBOX_API void SetShowingGrid(bool showing);
 
+    SANDBOX_API bool ManipulatorMouseWrap();
+    SANDBOX_API void SetManipulatorMouseWrap(const bool enabled);
```

Comment:
update/nit: Please remove the `const` here, it's not needed in the function interface when passed by value